.github/workflows/ci.yml

```
@@ -38,9 +43,16 @@ jobs:
           cache-dependency-path: '**/pnpm-lock.yaml'
       - run: pnpm install --frozen-lockfile
       - run: pnpm run compile
-      - name: Install Playwright's dependencies
+      - name: Cache Playwright
+        uses: actions/cache@v3
+        id: playwright-cache
+        with:
+          path: ~/.cache/ms-playwright
+          key: ${{ runner.os }}-playwright-1.40.1
```

```suggestion
          key: ${{ runner.os }}-${{ hashFiles('pnpm-lock.yaml') }}
```
should be good. https://github.com/actions/cache